README.md

```diff
@@ -109,7 +106,7 @@ cm.onConnect(function() {
 
 The *onNextConnect* method queues a task (the callback) to run the next time the device connects. If the imp is already connected, the callback will be invoked immediatly.
 
-The callback method takes zero parameters.
+*The callback method takes zero parameters.*
 
```

Comment:
How much data can be queued? Probably worth adding a note to be clear that the data is (I presume) held in memory within the class, and the amount you can store depends on what it is, the type of data, and what imp you're using?